site/es/_data/home.yml

```diff
@@ -153,7 +153,7 @@ developers:
           role: Head Engineer
       image:
         src: ix://lifestyle/developer_3.png
-        alt: Mike C. leaning against a desk.
+        alt: Mike C. recostado en un escritorio.
 community:
```

Comment:
Can you please take a look at this? [This](https://github.com/chromeos/chromeos.dev/blob/main/site/en/_data/home.yml#L132-L159) is the current content for it.